prime-router/src/main/kotlin/fhirengine/engine/CustomFhirPathFunctions.kt

```diff
@@ -144,4 +145,50 @@ class CustomFhirPathFunctions : FhirPathFunctions {
             mutableListOf(StringType(result))
         }
     }
+
+    /**
```

Comment:
FYI, some commented code that can get reverted

Reply:
All changes to this file should have been reverted